### DISCUSSION TITLE
My conditinals expressions are getting displaying in the HTML tag within the browser

### DISCUSSION THREAD

**Discussion by simo97** [adonisjs/core#4270] [Help]
Here is what i wrote in the edge temlate
```
@each(pCategory in parentCategories)
  <option value="{{category.id}}" @if(category.parentId == pCategory.id) selected @end>
      {{pCategory.title}}</option>
  @end
```

And here is what i got in the HTML templated rendered
<img width="734" alt="Screenshot 2023-10-13 at 15 18 50" src="https://github.com/adonisjs/core/assets/14660547/bdf9173a-e03a-4fc2-ba3c-39fcf5406ee1">

Is there a reason why it shows that condition the way it does in the browser? The condition seem to work by the way, but i am curious why it shows that.

**Comment by tomgobich** [comment]
Tags in EdgeJS must be on their own line. So, it's likely rendering like that because it doesn't recognize it as a tag.

Either of the below should work.
```
<!-- putting the if on a new line -->
@each(pCategory in parentCategories)
  <option
    id="basic-icon-default-company"
    @if (category.parentId == pCategory.id)
      selected
    @endif
  >
   {{ pCategory.title }}
  </option>
@endeach
```

```
<!-- using a ternary -->
@each(pCategory in parentCategories)
  <option id="basic-icon-default-company" {{ category.parentId == pCategory.id ? 'selected' : '' }}>
   {{ pCategory.title }}
  </option>
@endeach
```
